samples/msgext-search-auth-config/python/appManifest/manifest.json

```diff
@@ -2,7 +2,8 @@
   "$schema": "https://developer.microsoft.com/json-schemas/teams/v1.19/MicrosoftTeams.schema.json",
   "manifestVersion": "1.19",
   "version": "1.0.0",
-  "id": "<<YOUR-MICROSOFT-APP-ID>>",
+  "id": "${{TEAMS_APP_ID}}",
```

Comment:
icons not needed for appManifest?

Reply:
Need FYI
![image](https://github.com/user-attachments/assets/52a32c08-36eb-4e94-8415-60178718eedb)